management/email.go

```diff
@@ -19,6 +20,7 @@ type Email struct {
 }
 
 // EmailCredentials are used for authenticating Email Providers.
+// Deprecated: Use EmailProvider instead.
 type EmailCredentials struct {
 	// API User
 	APIUser *string `json:"api_user,omitempty"`
```

Comment:
To note that we're not adding the APIUser field to any of the credentials type within the new `EmailProvider` struct. This is because this field is no longer supported and trying to include it in the payload will result in a 400:

```
{
  "statusCode": 400,
  "error": "Bad Request",
  "message": "Payload validation error: 'None of the valid schemas were met' on property credentials (Credentials required to use the provider). Inner errors: [ Payload validation error: 'Additional properties not allowed: api_user' on property credentials ({description}). ].",
  "errorCode": "invalid_body"
}
```